Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
The dot should not be a problem. After all, many filenames include an extension, and we currently have a `README.md` target in the makefile. What's the error you're seeing?